csrc/host_ir/host_ir.cpp

```diff
@@ -71,7 +83,9 @@ NVFUSER_DEFINE_CLONE_AND_CREATE(PostOnStream)
 std::string PostOnStream::toString(int indent_size) const {
   int indent_increment = 2;
   std::stringstream ss;
-  indent(ss, indent_size) << "Execute the following Fusion, taking inputs :{\n";
+  indent(ss, indent_size) << "PostOnStream the operation :{\n";
```

Comment:
```suggestion
  indent(ss, indent_size) << "PostOnStream the operation :{" << std::endl;
```

I understand our codebase has been mixing \n and std::endl. @naoyam , what's your opinion here? Even though we don't serve Window users at this moment, the docker image we publish (including pytorch and apex etc) does support Windows. So a little bit of typing tax seems worthwhile for potential portability.

Reply:
What we print is mostly for informational debugging messages, so I think it's also good to use `std::endl` as it flushes the stream.